### ISSUE TITLE
Help wanted packaging for Linux distributions

### ISSUE THREAD

**Issue by hickford** [hickford/git-credential-oauth#21]
Help wanted packaging for Linux distributions. https://repology.org/project/git-credential-oauth

Top priority is Arch Linux

**Comment by hickford** [comment]
@shyim yes please. I started at https://github.com/NixOS/nixpkgs/pull/235073

**Comment by shyim** [comment]
https://github.com/NixOS/nixpkgs/pull/235464

**Comment by shyim** [comment]
Do you maybe want to link:
```
[![Packaging status](https://repology.org/badge/vertical-allrepos/git-credential-oauth.svg)](https://repology.org/project/git-credential-oauth/versions)
```
in the readme?

**Comment by hickford** [comment]
Good idea, done. https://github.com/hickford/git-credential-oauth/commit/a7fd2a4c2121b96fb218e3ebdc9c16a991b90048

**Comment by hickford** [comment]
Void Linux https://github.com/void-linux/void-packages/pull/44237

**Comment by bsteinb** [comment]
Are there any plans to publish packages to EPEL so `git-credential-oauth` can be installed conveniently on RHEL and its rebuilds? I noticed that [a SPEC file is there already](https://src.fedoraproject.org/rpms/git-credential-oauth/tree/epel9), but probably not hooked up to produce and publish RPMs.

I recently attempted to build that SPEC and had no luck on RHEL 9.4 due to issues in the RPM Go infrastructure. Those seem to have been fixed as of RHEL 9.5, however, and the SPEC file now builds successfully (with some small modifications to bring it in line with the SPEC files for more recent versions of `git-credential-oauth` in Fedora).

**Comment by hickford** [comment]
@bsteinb I'd like to but I don't have the necessary app `fedpkg` installed. Can you help?

* https://src.fedoraproject.org/rpms/git-credential-oauth and https://src.fedoraproject.org/rpms/golang-x-oauth2
* https://fedoraproject.org/wiki/SIGs/Go
* https://docs.fedoraproject.org/en-US/package-maintainers/Package_Update_Guide/
* https://docs.fedoraproject.org/en-US/package-maintainers/Joining_the_Package_Maintainers/

**Comment by bsteinb** [comment]
No, sorry. The `fedpkg` tool itself seems to only be a `dnf install` away, but that looks like the smallest step in becoming a package maintainer. Skimming the pages you linked it also involves getting multiple accounts to various services, signing up to mailing lists and introducing oneself to the packager community, getting sponsored, reading a bunch of guidelines, etc.

**Comment by skinchangerdev** [comment]
I've opened a PR for Void Linux void-linux/void-packages#58978, as the previous PR went stale due to a lack of a maintainer for the package.
